From 74bf2508e5cc46f72009f148edb9bf224e058e2d Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Wed, 19 Oct 2022 15:00:43 +0000
Subject: [PATCH 162/222] conf_native: Add --enable-gpl

(cherry picked from commit bab9bf4a2e39391940d88af2ce5d70236ac21f15)
(cherry picked from commit 3f949417e929ccbfab891fca97e4279b10e438e2)
---
 pi-util/conf_native.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pi-util/conf_native.sh b/pi-util/conf_native.sh
index 37cea71756..a9e053801c 100755
--- a/pi-util/conf_native.sh
+++ b/pi-util/conf_native.sh
@@ -93,6 +93,7 @@ $FFSRC/configure \
  --enable-libdrm\
  --enable-vout-egl\
  --enable-vout-drm\
+ --enable-gpl\
  $SHARED_LIBS\
  $RPIOPTS\
  --extra-cflags="-ggdb $RPI_KEEPS $RPI_DEFINES $RPI_INCLUDES"\
-- 
2.45.2

